packages/app-mobile/injectedJS.config.js

```diff
@@ -0,0 +1,25 @@
+// Configuration file for rollup
+
+const path = require('path');
```

Comment:
For new code, please only import what you need from packages. So `const { dirname } =require("path")`. This is if we ever implement tree shaking